Cogs/AdminCommands.py

```diff
@@ -84,13 +84,20 @@ async def status(self, ctx, *, status):
             status (str): Text to be displayed
         """
 
+        # open a file to store the status in
+        statusFile = open('assets/status.txt', 'w')
```

Comment:
This needs to be done with `async with aiofiles.open`... Synchronous file I/O in async function breaks things

Reply:
Look in `log()` function for example of how to do this